### PR TITLE
Fixes tgui Image component pixelation

### PR DESCRIPTION
**Pull request by TobleroneSwordfish**
[BUG] [UI]
## About the PR 
The `pixelated` property on `Image` didn't actually work, now it does.

## Why's this needed? 
`<Image pixelated />` is much nicer than ``

### CODE REVIEW

File: tgui/packages/tgui/components/Image.js [hunk 1]
```diff
@@ -4,9 +4,11 @@ import { Box } from "./Box";
 export const Image = (props) => {
   const { pixelated, className, ...rest } = props;
 
-  return (<Box as="img" {...rest} className={classes(
-    "Image",
-    pixelated && "Image--pixelated",
```
Comment (ZeWaka) [inline tgui/packages/tgui/components/Image.js:9]: why not just fix the css?

Comment (ZeWaka) [reply tgui/packages/tgui/components/Image.js:9]: since the talk with style, since apparently fixing the css doesn't work - just remove the css portion

**Review by ZeWaka — COMMENTED** [review]
sorry to ask again but can we add Image to
https://github.com/goonstation/goonstation/blob/master/tgui/docs/component-reference.md

**Comment by TobleroneSwordfish** [comment]
!merge_upstream

**Pull request merged**